src/components/IssueQueue.vue

```diff
@@ -44,7 +44,8 @@ const filteredStates = computed(() => {
     .filter(
       (state) =>
         state !== FlawClassificationStateEnum.Done
-        && state !== FlawClassificationStateEnum.Rejected,
+        && state !== FlawClassificationStateEnum.Rejected
+        && state !== FlawClassificationStateEnum.Empty,
     )
     .join(',');
 });
```

Comment:
Theoretically we shouldn't need this, but I guess the question wether or not we want users (and us) to able to easily know if OSIDB is returning something it shouldn't